graphqlapi/build_schema.go

```diff
@@ -0,0 +1,277 @@
+	}
+
+	return rootFields, nil
+}
+
+// generate the static parts of the schema
+func (g *GraphQL) genThingsAndActionsFieldsForWeaviateLocalConvertedFetchObj(localConvertedFetchActions *graphql.Object, localConvertedFetchThings *graphql.Object) *graphql.Object {
+	convertedFetchThingsAndActionFields := graphql.Fields{
+
+		"Actions": &graphql.Field{
+			Name:        "WeaviateLocalConvertedFetchActions",
+			Description: "Locate Actions on the local Weaviate",
+			Type:        localConvertedFetchActions,
+			Resolve: func(p graphql.ResolveParams) (interface{}, error) {
+				return nil, fmt.Errorf("Not supported")
+			},
+		},
+
+		"Things": &graphql.Field{
+			Name:        "WeaviateLocalConvertedFetchThings",
+			Description: "Locate Things on the local Weaviate",
+			Type:        localConvertedFetchThings,
+			Resolve: func(p graphql.ResolveParams) (interface{}, error) {
+				return nil, fmt.Errorf("Not supported")
+			},
+		},
+	}
+
+	convertedFetchThingsAndActionFieldsObject := graphql.ObjectConfig{
+		Name:        "WeaviateLocalConvertedFetchObj",
+		Fields:      convertedFetchThingsAndActionFields,
+		Description: "Fetch things or actions on the internal Weaviate",
+	}
+
+	return graphql.NewObject(convertedFetchThingsAndActionFieldsObject)
+}
+
+func (g *GraphQL) genThingsAndActionsFieldsForWeaviateLocalMetaFetchGenericsObj(localMetaFetchActions *graphql.Object, localMetaFetchThings *graphql.Object) *graphql.Object {
+	metaFetchGenericsThingsAndActionFields := graphql.Fields{
+
+		"Actions": &graphql.Field{
+			Name:        "WeaviateLocalMetaFetchGenericsActions",
+			Description: "Actions to fetch for meta generic fetch",
+			Type:        localMetaFetchActions,
+			Args: graphql.FieldConfigArgument{
+				"_maxArraySize": &graphql.ArgumentConfig{
+					Description: "If there are arrays in the result, limit them to this size",
+					Type:        graphql.Int,
+				},
+			},
+			Resolve: func(p graphql.ResolveParams) (interface{}, error) {
+				return nil, fmt.Errorf("Not supported")
+			},
+		},
+
+		"Things": &graphql.Field{
+			Name:        "WeaviateLocalMetaFetchGenericsThings",
+			Description: "Things to fetch for meta generic fetch",
+			Type:        localMetaFetchThings,
+			Args: graphql.FieldConfigArgument{
+				"_maxArraySize": &graphql.ArgumentConfig{
+					Description: "If there are arrays in the result, limit them to this size",
+					Type:        graphql.Int,
+				},
+			},
+			Resolve: func(p graphql.ResolveParams) (interface{}, error) {
+				return nil, fmt.Errorf("Not supported")
+			},
+		},
+	}
+
+	metaFetchGenericsThingsAndActionFieldsObject := graphql.ObjectConfig{
+		Name:        "WeaviateLocalMetaFetchGenericsObj",
+		Fields:      metaFetchGenericsThingsAndActionFields,
+		Description: "Object type to fetch",
+	}
+
+	return graphql.NewObject(metaFetchGenericsThingsAndActionFieldsObject)
+}
+
+func (g *GraphQL) genGenericsFieldForWeaviateLocalMetaFetchObj(localMetaFetchObject *graphql.Object) *graphql.Object {
```

Comment:
Hmm. I'm wondering if it is necessary to define this function on `*Graphql`. I think it could be a stand-alone function; `g` is not used in this function at all.